packages/node/src/dev-server.ts

```diff
@@ -81,8 +81,9 @@ register({
 
 import { createServer, Server, IncomingMessage, ServerResponse } from 'http';
 import { Readable } from 'stream';
-import { Bridge } from './bridge';
-import { getNowLauncher } from './launcher';
+import type { Bridge } from '@vercel/node-bridge/bridge';
+// @ts-ignore - copied to the `dist` output as-is
```

Comment:
Are we able to remove the `@ts-ignore` If we copy over the `launcher.d.ts` file?

Reply:
It gets tricky because we need to copy over multiple type files.

I'll see if this can be improved in a follow up PR 👍